### PR TITLE
SPOI-10491 Properties changes for following apps as per new support f…

### PR DESCRIPTION
**Pull request by deepak-narkhede**
SPOI-10491 Properties changes for below apps as per new support for app Launch properties
    
    1) HDFS Sync App.
    2) HDFS line copy App.
    3) HDFS to Kafka Sync App.
    4) HDFS to S3 Sync App.
    5) Database to Database Sync.
    6) Database to HDFS App.
    7) Kafka to Database Sync App.
    8) Kafka to HDFS Filter App.
    9) Kafka to HDFS Sync App.
    10) S3 to HDFS Sync App.

Tested with RTS 3.7.1 package

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@yogidevendra .Please review

File: database-to-hdfs/src/main/resources/META-INF/properties.xml [hunk 1]
```diff
@@ -19,46 +19,249 @@
...
+  </property>
+  <property>
+    <name>dt.operator.JdbcPoller.prop.tableName</name>
+    <value>${apex.app-param.jdbcInputTableName}</value>
+  </property>
 
   <property>
+    <name>apex.app-param.whereCondition</name>
+    <value></value>
+    <description>Specify WHERE clause for input database.</description>
```
Comment (deepak-narkhede) [inline database-to-hdfs/src/main/resources/META-INF/properties.xml]: Done.

File: database-to-hdfs/src/main/resources/META-INF/properties.xml [hunk 2]
```diff
@@ -19,46 +19,249 @@
...
+    <name>apex.app-param.pollIntervalForJdbcInput</name>
+    <value>1000</value>
+    <description>Specify the poll interval for Jdbc input.</description>
+  </property>
+  <property>
+    <name>dt.operator.JdbcPoller.prop.pollInterval</name>
+    <value>${apex.app-param.pollIntervalForJdbcInput}</value>
+  </property>
+
+  <property>
```
Comment (deepak-narkhede) [inline database-to-hdfs/src/main/resources/META-INF/properties.xml]: Done.

File: database-to-hdfs/src/main/resources/META-INF/properties.xml [hunk 3]
```diff
@@ -19,46 +19,249 @@
...
+    <name>apex.app-param.tupleClassNameForTransformOutput</name>
+    <value>com.datatorrent.apps.PojoEvent</value>
+    <description>Specify tuple class name for transform output.</description>
+  </property>
+  <property>
+    <name>dt.operator.transform.port.output.attr.TUPLE_CLASS</name>
+    <value>${apex.app-param.tupleClassNameForTransformOutput}</value>
   </property>
 
 </configuration>
```
Comment (deepak-narkhede) [inline database-to-hdfs/src/main/resources/META-INF/properties.xml]: Done.

**Comment by deepak-narkhede** [comment]
@chaithu14. Incorporated version changes and comments.

**Comment by chaithu14** [comment]
@deepak-narkhede Version change is not required for kinesis-to-s3 and hdfs-hdfs-transform apps.

**Comment by deepak-narkhede** [comment]
@chaithu14 . I think version changes are required for kinesis-to-s3 and hdfs-hdfs-transform as they also have changes with new property format.

**Comment by chaithu14** [comment]
I think the version should be 0.8 as kinesis-to-s3 and hdfs-hdfs-transform are new apps. @yogidevendra @ashwinchandrap Could you please confirm on this.

**Comment by chaithu14** [comment]
Looks good to me.

**Pull request merged**